openmm_septop/python_tutorial.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>Link to your paper for more details, people might be less familiar with this protocol. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>